__tests__/test-utils.js

```diff
@@ -198,7 +198,8 @@ class TestUtils {
 
   static startMocks() {
     // http mocks
-    httpMock.onGet('/version').reply(200, httpFixtures['/v1a/version']);
```

Comment:
Why did you have to change this?

Reply:
I didn't have to. I just thought it would be clearer to add the full address.

The way these mocks are declared make it confusing to know whether these endpoints being mocked belong to wallet-headless itself, to the fullnode or to tx-mining-service.

Also to keep the same pattern as I used for the `/health` mock on the fullnode. For it I necessarily had to add the fullnode address, because the wallet-headless also has a `/health` endpoint.

We could undo the change for `/version`, though. Tests will keep working.